docs/user/apirule-migration/01-83-migrate-jwt-v1beta1-to-v2.md

```diff
@@ -0,0 +1,154 @@
+    ```yaml
+    host: httpbin.local.kyma.dev
+    service:
+      name: httpbin
+      namespace: test
+      port: 8000
+    gateway: kyma-system/kyma-gateway
+    rules:
+      - path: /anything
+        methods:
+          - POST
+        accessStrategies:
+          - handler: jwt
+            config:
+              jwks_urls:
+                -  https://{IAS_TENANT}.accounts.ondemand.com/oauth2/certs
+      - path: /.*
+        methods:
+          - GET
+        accessStrategies:
+          - handler: jwt
+            config:
+              jwks_urls:
+                -  https://{IAS_TENANT}.accounts.ondemand.com/oauth2/certs
+    ```
+    The above configuration uses the **jwt** handler to expose the HTTPBin service's `/anything` and `/.*` endpoints.
+
+2. Adjust the obtained configuration to align with the **jwt** configuration of APIRule `v2`. 
+
+    To ensure the APIRule specification is compatible with version `v2`, you must include a mandatory field named **issuer** in the **jwt** handler's configuration.
+    You can find the **issuer** URL in the OIDC well-known configuration of your tenant, located at `https://{YOUR_TENANT}.accounts.ondemand.com/.well-known/openid-configuration`. Additionally, note that the value of the `jwks_urls` field is now stored in the `jwksUri` field.  Tokens do not need to be reissued unless they have expired. 
+    See an example of the adjusted APIRule configuration for version `v2`:
+
+    ```yaml
+    apiVersion: gateway.kyma-project.io/v2
+    kind: APIRule
+    metadata:
+      name: httpbin
+      namespace: test
+    spec:
+      hosts:
+        - httpbin
+      service:
+        name: httpbin
+        namespace: test
+        port: 8000
+      gateway: kyma-system/kyma-gateway
+      rules:
+        - jwt:
+            authentications:
+              -  issuer: https://{YOUR_TENANT}.accounts.ondemand.com
+                jwksUri: https://{YOUR_TENANT}.accounts.ondemand.com/oauth2/certs
+          methods:
+            - POST
+          path: /anything
+        - jwt:
+            authentications:
+              -  issuer: https://{YOUR_TENANT}.accounts.ondemand.com
+                jwksUri: https://{YOUR_TENANT}.accounts.ondemand.com/oauth2/certs
+          methods:
+            - GET
+          path: /{**}
+    ```
+    > [!NOTE]
+    > Note that the **hosts** field accepts a short host name (without a domain). Additionally, the path `/.*` has been changed to `/{**}` because APIRule `v2` does not support regular expressions in the **spec.rules.path** field.
+    >
+    > For more information, see the [Changes Introduced in APIRule `v2`](../custom-resources/apirule/04-70-changes-in-apirule-v2.md) document. **Read this document before applying the new APIRule `v2`.**
```

Comment:
```suggestion
    > For more information, see [Changes Introduced in APIRule `v2`](../custom-resources/apirule/04-70-changes-in-apirule-v2.md).
```
Maybe reading this document should be a prerequisite instead?

Edit: Added to line 11.

Reply:
good idea, thanks